docs/pages/management/operations/ca-rotation.mdx

```diff
@@ -24,7 +24,8 @@ Teleport signs Windows Desktop certificates with the user certificate authority.
 If the user CA is rotated, the new CA certificate will need to be exported and
 configured in group policy.
 
-[Read more about exporting the Teleport CA](../../desktop-access/active-directory-manual.mdx#export-the-teleport-ca)
+For more information about exporting the Teleport CA, see [Configure a GPO to allow Teleport 
```

Comment:
I would remove this change.

CA rotations are a large topic that applies to all Teleport resources, and configuring a GPO is very specific to Windows desktops.

Reply:
So, you would remove the cross-reference altogether?

Reply:
Desktop page can reference the CA rotation page, I just wouldn't link back from CA rotation to desktop.